src/main/java/com/zufar/onlinestore/cart/api/ItemsTotalPriceCalculator.java

```diff
@@ -1,17 +1,18 @@
 package com.zufar.onlinestore.cart.api;
 
-import com.zufar.onlinestore.cart.dto.ShoppingSessionItemDto;
+import com.zufar.onlinestore.cart.entity.ShoppingSessionItem;
+import org.mapstruct.Named;
 import org.springframework.stereotype.Service;
 import java.math.BigDecimal;
-import java.util.List;
+import java.util.Set;
 
 @Service
 public class ItemsTotalPriceCalculator {
 
-    public BigDecimal calculate(List<ShoppingSessionItemDto> items) {
+    @Named("toItemsTotalPrice")
```

Comment:
why do we need this annotation here

Reply:
![image](https://github.com/Sunagatov/Online-Store/assets/84774469/4d8060fd-be76-41c1-b827-a95f8f570228)